app/src/main/res/drawable/ic_alarm_grey_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#797979"
```

Comment:
Vector icons color can change at runtime, remove the word `_grey `from all the icons names.
For consistency reasons set the icons tint to `#000000`.